src/celestia/url.cpp

```diff
@@ -287,9 +287,9 @@ Url::goTo()
     // Position and orientation stored in frame coordinates; convert them
     // to universal and set the observer position.
     double tdb = sim->getTime();
-    auto coord = sim->getObserver().getFrame()->convertToUniversal(m_state.m_observerPosition, m_state.m_tdb);
+    auto coord = sim->getObserver().getFrame()->convertToUniversal(m_state.m_observerPosition, tdb);
```

Comment:
Original use of `m_state.m_tdb` here was probably a bug?

Reply:
yes, it should be used only when m_timeSource == UseUrlTime